src/sema/tags.rs

```diff
@@ -13,7 +13,7 @@ pub fn resolve_tags(
 ) -> Vec<Tag> {
     let mut res: Vec<Tag> = Vec::new();
 
-    for c in doc.iter().flat_map(pt::DocComment::comments) {
+    'next_doc: for c in doc.iter().flat_map(pt::DocComment::comments) {
```

Comment:
This label makes the code very unreadable, because you continue the loop from down below. Is there any way to make it cleaner?

Reply:
You're right, this can be written without labels. I've re-written without a for loop (find on iterator)